src/libs/OptionsListUtils.js

```diff
@@ -696,6 +697,11 @@ function getOptions(
         }
         const isSearchingSomeonesPolicyExpenseChat = !report.isOwnPolicyExpenseChat && searchValue !== '';
 
+        // When passing includeP2P false we are trying to hide features from users that are not ready for P2P and limited to workspace chats only.
+        if (!includeP2P && !isPolicyExpenseChat) {
+            return;
+        }
+
```

Comment:
NAB: Can we move this up with the other early returns? Or ever right below the first early return since that one seems similar?

Reply:
Where exactly? I'm not sure what change you want to see.

Reply:
Maybe here?

```js
if (isPolicyExpenseChat && report.isOwnPolicyExpenseChat && !includeOwnedWorkspaceChats) {
```